btree/btree.c

```diff
@@ -4395,7 +4505,7 @@ static void m0_btree_ut_basic_tree_oper(void)
 	struct m0_btree_op      b_op = {};
 	void                   *temp_node;
 	const struct node_type *nt = &fixed_format;
-
+	int                     rc;
 	/** Prepare transaction to capture tree operations. */
```

Comment:
Add a blank line after variable declarations.

Reply:
Ok. I will update it.